TACC/Core-Portal
# Task/fp 120 add file selector

## Overview: ##

This PR 
* adds file selector dialog (accessible on app page)
* makes various ui/styling changes to copy/move dialogs

## PR Status: ##


## Related Jira tickets: ##

* [FP-120](https://jira.tacc.utexas.edu/browse/FP-120)
* [FP-453](https://jira.tacc.utexas.edu/browse/FP-453) (design)

## Summary of Changes: ##

- Add select modal
- Update copy/move modals to match latest design (https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/screen/85634389-b55c-4166-9ccd-521d56777522)
-- Update text and spacing on modals
-- Remove table headers in selected files table
-- Remove bottom buttons
-- Ensure last part of breadcrumb should not be a link
-- Updates font sizes

## Testing Steps: ##
**Test file/folder selection**
1. Ensure you have apps (e.g. namd or rstudio) which allow you to select files.  If not, update your settings secret (`_PORTAL_APPS_METADATA_NAMES = ["frontera_apps", "frontera_apps_dev"]`)
2. Go to app that allows you to select files. 
3. Click on file/folder input to see 
4. Navigate through your folders and confirm Back button works and that you can select whatever current subfolder you are in.

**Test copy/move modals**
1. Go to Data Files, select some files and click copy or move to start using the Copy/Move dialogs 
2. Confirm they have changes outlined in [FP-453](https://jira.tacc.utexas.edu/browse/FP-453) + [design mockups](https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/screen/85634389-b55c-4166-9ccd-521d56777522).  (except for ([FP-540](https://jira.tacc.utexas.edu/browse/FP-540))  and ([FP-541](https://jira.tacc.utexas.edu/browse/FP-541)) 
3. Confirm that dialog works as it dif before
## UI Photos:

![Screen Shot 2020-07-27 at 4 59 31 PM](https://user-images.githubusercontent.com/8287580/88596633-215aee80-d02b-11ea-83b8-f9516dbf2186.png)
![Screen Shot 2020-07-27 at 4 59 39 PM](https://user-images.githubusercontent.com/8287580/88596631-20c25800-d02b-11ea-8ddb-d81947d56427.png)
![Screen Shot 2020-07-27 at 5 00 07 PM](https://user-images.githubusercontent.com/8287580/88596629-2029c180-d02b-11ea-86fe-7b824d2b9e92.png)
![Screen Shot 2020-07-27 at 4 59 57 PM](https://user-images.githubusercontent.com/8287580/88596630-20c25800-d02b-11ea-86be-2754ed544dda.png)




## Notes: ##
- We don't allow the user to select the top level/root folder in Select Modal but we do allow the user to choose the top-level/root folder in Copy/Move modals
- A related task is https://jira.tacc.utexas.edu/browse/FP-218 where we will support different tapis systems. That work will be done separately. 
- Created JIra ticket ([FP-540](https://jira.tacc.utexas.edu/browse/FP-540)) for "New Directory"  button in Copy/Move modals. See https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/screen/85634389-b55c-4166-9ccd-521d56777522/ *
- Created JIra ticket ([FP-541](https://jira.tacc.utexas.edu/browse/FP-541)) for remove button in Copy/Move modals. See https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/screen/85634389-b55c-4166-9ccd-521d56777522/ * 

* thanks Wes
